attributes/default.rb

```diff
@@ -5,8 +5,8 @@
 default["kafka"]["user"] = "kafka"
 default["kafka"]["group"] = "kafka"
 
-default["kafka"]["scala_version"] = "2.9.2"
-default["kafka"]["version"] = "0.8.1.1"
+default["kafka"]["scala_version"] = "2.11"
+default["kafka"]["version"] = "0.9.0.0"
```

Comment:
Update README @ https://github.com/cerner/cerner_kafka#how-to-specify-where-to-download-kafka-from-and-which-version-to-install